BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,313 @@
+    WebElement ingredientsSection;
+
+    @FindBy(css = "#mini-panel-product_ingredients_modal h2")
+    WebElement ingredientsHeading;
+
+    @FindBy(css = "li #used-for")
+    WebElement usedForSection;
+
+    @FindBy(id = "tab_used-for")
+    WebElement usedForHeading;
+
+    @FindBy(css = "li #warnings")
+    WebElement warningsSection;
+
+    @FindBy(id = "tab_warnings")
+    WebElement warningsHeading;
+
+    @FindBy(id = "mini-panel-product_faq")
+    WebElement faqsSection;
+
+    @FindBy(css = "#mini-panel-product_faq span")
+    WebElement faqsHeading;
+
+    @FindBy(css = "#bv-product-reviews")
+    WebElement reviewsSection;
+	
+	@FindBy(xpath = "//div[@id='onetrust-close-btn-container']")
+	WebElement closePrivacyBtn;
+    
+    public WebElement getJumpToHeadings(String headingText) {
+        return driver.findElement(By.xpath("//div[@class='jump-to-wrapper']//ul//li//a[text()='" + headingText + "']"));
+    }
+	
+	/**
+	 * Function to click on close privacy pop-up button
+	 */
+	public void closePrivacyPopup() {
+		Action.explicitWaitForElementTobeclickable(closePrivacyBtn, 30);
+		boolean eleDisplayed = closePrivacyBtn.isDisplayed();
+		if(eleDisplayed) {
+			extentPassLog("Privacy pop-up displayed : ", true);
+			Action.performActionwithExtentInfoLog(closePrivacyBtn, "click", "Clicking on : Close button");
+		} else extentFailLog("Privacy pop-up displayed : ", false);
+	}
+
+    /**
+     * Visits a product page from the homepage.
+     */
+    public void visitPDP() {
+        Action.performActionwithExtentInfoLog(prodHeaderMenu, "click", "Clicking on : " + prodHeaderMenu.getText());    
+        Action.performActionwithExtentInfoLog(prodTileTitle, "click", "Clicking on : " + prodTileTitle.getText()); 	
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies if the image of the product is visible.
+     */
+    public void verifyProdImage() {
+    	this.visitPDP();
+        extentInfoLog("Image is displayed : ", Action.isDisplayed(driver, prodImg));
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies the title of the product.
+     */
+    public void verifyProdTitle() {
+    	this.visitPDP();
+        extentInfoLog("Title is displayed : ", Action.isDisplayed(driver, prodTitle));
+        extentInfoLog("Product title verified");
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies if the rating of the product is visible.
+     */
+    public void verifyProdRating() {
+    	this.visitPDP();
+        extentInfoLog("Rating is displayed : ", Action.isDisplayed(driver, prodRating));
+    }
+
+    /**
+     * Visits a product page from the homepage.
+     * Then, verifies if the overview of the product is visible.
+     */
+    public void verifyProdOverview() {
+    	this.visitPDP();
+        extentInfoLog("Overview is displayed : ", Action.isDisplayed(driver, prodOverview));
```

Comment:
@zahinSJI assertion?? just checking isdisplayed is not acceptable. i want assertion for each testcase.

Reply:
Assertion added